package.json

```diff
@@ -25,15 +26,18 @@
     "metal"
   ],
   "dependencies": {
+    "bootstrap": "^3.3.6",
     "metal": "^2.0.0",
     "metal-state": "^2.0.0",
     "metal-dom": "^2.0.0"
   },
   "devDependencies": {
     "babel-cli": "^6.4.5",
+    "babel-loader": "^7.1.2",
+    "babel-plugin-transform-node-env-inline": "^0.2.0",
```

Comment:
Do we need this one? If we don't have incremental-dom I think this is not needed.

Reply:
Yes, you are right, I'm addressing this now.